core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -89,14 +104,29 @@ oppia.factory('explorationData', [
           // changes applied. This makes a force-refresh necessary when changes
           // are discarded, otherwise the exploration-with-draft-changes
           // (which is cached here) will be reused.
+          console.log(errorCallback);
           return (
             EditableExplorationBackendApiService.fetchApplyDraftExploration(
             explorationId).then(function(response) {
               $log.info('Retrieved exploration data.');
               $log.info(response);
               draftChangeListId = response.draft_change_list_id;
               explorationData.data = response;
-
+              var draftChanges = LocalStorageService.getExplorationDraft(
```

Comment:
Sure. I've gotten started on that, just need to edit it a little so should send another push today to finish that up.